sdk/python/feast/batch_feature_view.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         *,
         name: str,
         source: DataSource,
-        entities: Optional[Union[List[Entity], List[str]]] = None,
```

Comment:
This seems to be u related to usage fyi

Reply:
I know, after I removed decorators, a lot of linter rules started to fail and I had to fix a bunch of them (~20). Probably something with mypy and decorators not getting along, not sure why it wasn't an issue before.

Reply:
Nice, can you add that to the description and PR notes?